source/logging.c

```diff
@@ -547,8 +547,13 @@ int aws_logger_init_noalloc(
         impl->file = options->file;
         impl->should_close = false;
     } else { /* _MSC_VER */
-        impl->file = aws_fopen(options->filename, "w");
-        impl->should_close = true;
+        if (options->filename != NULL) {
```

Comment:
unrelated: I have no idea what this `/* _MSC_VER */` is about

Reply:
Yeah, I just left it because it was there. I'm guessing that maybe it was to fix a MSVC issue at one point?